demos/common/performance_metrics/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+find_package(InferenceEngine 2.0 REQUIRED)
```

Comment:
Why do you need IE?

Reply:
performance_metrics.hpp includes ocv_common.hpp, which needs ie_core.hpp.

I've moved `find_package(InferenceEngine 2.0 REQUIRED)` to CMakeLists file for the whole `common` library, since the folder structure in `common` was modified.